articles/cognitive-services/cognitive-services-development-options.md

```diff
@@ -111,11 +111,11 @@ You can use Azure DevOps and GitHub actions to manage your deployments. In the [
 
 As you progress on your journey building an application or workflow with the Cognitive Services, you may find that you need to customize the model to achieve the desired performance. Many of our services allow you to build on top of the pre-built models to meet your specific business needs. For all our customizable services, we provide both a UI-driven experience for walking through the process as well as APIs for code-driven training. For example:
 
-* You want to train a Custom Speech model to correctly recognize medical terms with a word error rate (WER) below 3%
+* You want to train a Custom Speech model to correctly recognize medical terms with a word error rate (WER) below 3 percent
```

Comment:
I think we need to revert this to 3$ based on the MS Style Guide entry: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/percent-percentage

Reply:
The style guide mentions that we should use the percent sign (%) only if there is a space constraint. However, you can take a call on this.

Reply:
I am curious how this is handled (and if it's handled correctly) in localization. But I won't block.